src/carcockpit/shaders/shader_adv.cpp

```diff
@@ -190,6 +172,7 @@ void shader_adv::render(
 	const ruis::render::texture_2d& tex_color,
```

Comment:
`projection` argument seems to be unused, remove it

Reply:
done